Library/RemoteConfig/RemoteConfigFeature+HelpersTests.swift

```diff
@@ -3,111 +3,102 @@ import Prelude
 import XCTest
 
 final class RemoteConfigFeatureHelpersTests: TestCase {
-  func testBlockUsers_RemoteConfig_FeatureFlag_False() {
+  func assertFeatureIsFalse(_ featureFlag: Bool,
+                            whenRemoteConfigFeatureIsFalse feature: RemoteConfigFeature) {
```

Comment:
Normally I'm fine with boilerplate in tests, but in this case, I think the boilerplate was impeding clarity.